src/utils/mute_grad.jl

```diff
@@ -0,0 +1,79 @@
+
+
+function mutearoundmultiplepoints!(inparr::Array{T,N},xyzpts::Matrix{T},grid::UniformFiniteDifferenceGrid{N,T},
+                                   radiuspx::Integer) where {T,N}
+    for i=1:size(xyzpts,1)
+        mutearoundpoint!(inparr,xyzpts[i,:],grid,radiuspx)
+    end
+    return
+end
+
+
+
+function mutearoundpoint!(inparr::Array{T,N},xyzpt::Vector{T},grid::UniformFiniteDifferenceGrid{N,T},
+                          radiuspx::Integer) where {T,N}
+
+    @assert size(inparr) == grid.size
+    ety = eltype(inparr)
+    Ndim = ndims(inparr)
+    rmax::T = radiuspx*maximum(NTuple(grid.spacing))
+
+    # This is just for the future, in case an offset of the grid
+    #   coordinates could be specified
+    #gridinit = @SVector zeros(T,Ndim)
+
+    ijkmin = MVector{Ndim,Int64}(undef)
+    ijkmax = MVector{Ndim,Int64}(undef)
+    for d=1:Ndim
+        # check that the point is inside the grid
+        # with offset
+        # if gridinit[d] <= xyzpt[d] <= grid.extent[d]
```

Comment:
Remove unused code in comments

Reply:
Disagree, sorry. It's for a potential change in the future.